neqo-transport/src/connection/mod.rs

```diff
@@ -1142,6 +1153,25 @@ impl Connection {
 
         match (&self.state, self.role) {
             (State::Init, Role::Client) => {
+                // Before starting the client, send some noise
+                if self.conn_params.pre_init_noise_enabled() && self.pre_initial_pkts > 0 {
+                    self.pre_initial_pkts -= 1;
+                    if let Some(path) = self.paths.primary() {
+                        let v = random::<1>()[0] as usize;
+                        let len = PRE_INIT_PAYLOAD_LEN_RANGE.0 + (v % (PRE_INIT_PAYLOAD_LEN_RANGE.1 - PRE_INIT_PAYLOAD_LEN_RANGE.0 + 1));
```

Comment:
Can this be simplified or at least explained better? `Range` may have some helper functions you can use.